docs/concepts/gateway/customization.md

```diff
@@ -224,7 +224,7 @@ class MyGateway(FastAPIBaseGateway):
         return app
 ```
 
-### Recovering executor error
+### Recovering Executor error
```

Comment:
Should this be singular or plural?

Reply:
Depends on how the user batches the input docs. If there is not batching then there will be only one otherwise it will match the batch size.

Reply:
Let's make it plural. It sounds more natural